CONTRIBUTING.md

```diff
@@ -29,22 +29,43 @@ Thanks for your interest in contributing to the Stencil Docs! :tada:
 2. Clone your fork.
 3. Make a branch for your change.
 4. Run `npm install` (make sure you have [node](https://nodejs.org/en/) and [npm](http://blog.npmjs.org/post/85484771375/how-to-install-npm) installed first).
-5. Run `npm run docs` to build the documentation files.
-6. Run `npm start`.
+5. Run `npm run build` to build the documentation files.
+6. Run `npm run serve` to serve the built site.
```

Comment:
This works, but you can also just run `npm run start`

Reply:
True, although [anchor links don't necessarily work with `npm run start`.](https://docusaurus.io/docs/cli#docusaurus-start-sitedir) I'll update this to point to `npm start`, with the caveat that if anchor links are included, folks should run `build`/`serve`

Reply:
https://github.com/ionic-team/stencil-site/pull/1052/commits/e99b2ab8bab7664b485cff4a0b34214dba0b14c0